h5p-bildetema-topic-list/src/components/App/App.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
-import { Topic } from "../../../../types";
-import { fetchData, getTopics } from "../../utils/data.utils";
+import { Topic } from "../../../../common/types/types";
```

Comment:
I'm not a huge fan, but we might consider adding an alias to the `common` directory to remove the `../../../../` https://dev.to/larswaechter/path-aliases-with-typescript-in-nodejs-4353